### ISSUE TITLE
Various errors with ember 2.9

### ISSUE THREAD

**Issue by jonblack** [open-tux/ember-bulma#44]
I'm using ember 2.9 with node 6.9.1. I've followed the instructions to install and import ember-bulma. When I run the development server (`ember s`) I get the following errors:

```
Livereload server on http://localhost:49152
Serving on http://localhost:4200/

modules/ember-bulma/components/bulma-button.js: line 29, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-button.js: line 38, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-button.js: line 70, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-button.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-button.js: line 3, col 20, 'readOnly' is defined but never used.
modules/ember-bulma/components/bulma-button.js: line 3, col 30, 'alias' is defined but never used.
modules/ember-bulma/components/bulma-button.js: line 3, col 37, 'bool' is defined but never used.

7 errors

modules/ember-bulma/components/bulma-checkbox.js: line 7, col 3, 'set' is defined but never used.

1 error

modules/ember-bulma/components/bulma-column.js: line 3, col 8, 'computed' is defined but never used.

1 error

modules/ember-bulma/components/bulma-hero-content.js: line 34, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-hero-content.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-hero-content.js: line 3, col 20, 'alias' is defined but never used.

3 errors

modules/ember-bulma/components/bulma-hero.js: line 49, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-hero.js: line 4, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-hero.js: line 4, col 20, 'alias' is defined but never used.

3 errors

modules/ember-bulma/components/bulma-input.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-input.js: line 3, col 20, 'alias' is defined but never used.
modules/ember-bulma/components/bulma-input.js: line 4, col 10, 'makeString' is defined but never used.
modules/ember-bulma/components/bulma-input.js: line 9, col 3, 'set' is defined but never used.

4 errors

modules/ember-bulma/components/bulma-message.js: line 18, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-message.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-message.js: line 3, col 20, 'readOnly' is defined but never used.
modules/ember-bulma/components/bulma-message.js: line 3, col 30, 'alias' is defined but never used.

4 errors

modules/ember-bulma/components/bulma-modal-close.js: line 6, col 3, 'set' is defined but never used.

1 error

modules/ember-bulma/components/bulma-modal.js: line 8, col 3, 'set' is defined but never used.

1 error

modules/ember-bulma/components/bulma-notification.js: line 18, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-notification.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-notification.js: line 3, col 20, 'readOnly' is defined but never used.

3 errors

modules/ember-bulma/components/bulma-radio.js: line 7, col 3, 'set' is defined but never used.

1 error

modules/ember-bulma/components/bulma-section.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-section.js: line 3, col 20, 'alias' is defined but never used.

2 errors

modules/ember-bulma/components/bulma-select.js: line 31, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-select.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-select.js: line 3, col 20, 'readOnly' is defined but never used.

3 errors

modules/ember-bulma/components/bulma-tabs.js: line 45, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-tabs.js: line 66, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-tabs.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-tabs.js: line 3, col 20, 'readOnly' is defined but never used.
modules/ember-bulma/components/bulma-tabs.js: line 3, col 30, 'alias' is defined but never used.
modules/ember-bulma/components/bulma-tabs.js: line 4, col 10, 'makeString' is defined but never used.

6 errors

modules/ember-bulma/components/bulma-tag.js: line 27, col 3, Unexpected '@'.
modules/ember-bulma/components/bulma-tag.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-tag.js: line 3, col 20, 'readOnly' is defined but never used.

3 errors

modules/ember-bulma/components/bulma-textarea.js: line 3, col 8, 'computed' is defined but never used.
modules/ember-bulma/components/bulma-textarea.js: line 3, col 20, 'read' is defined but never used.
modules/ember-bulma/components/bulma-textarea.js: line 7, col 3, 'get' is defined but never used.
modules/ember-bulma/components/bulma-textarea.js: line 8, col 3, 'set' is defined but never used.

4 errors

modules/ember-bulma/utils.js: line 58, col 2, Missing semicolon.
modules/ember-bulma/utils.js: line 5, col 3, 'A' is defined but never used.
```

I'm still able to use bulma by writing raw HTML, but some components (perhaps all of them) don't work.

**Comment by crodriguez1a** [comment]
@jonblack Hm, these look more like linting errors than anything blocking. Do see any specific errors when trying to render a component?  Also, could you provide your package.json?

cc/ @jbailey4 @alexdiliberto

**Comment by jonblack** [comment]
Yes, you're right, it's just lint errors. I don't recall errors when rendering, but there's a change I didn't get that far. I stopped using it because lots of tests also fail because of the lint errors, and it was distracting. Sorry I'm not that helpful.

**Comment by jbailey4** [comment]
@jonblack Yeah this is cleanup we need to do in order to remove those linting errors. I can add this to top of my list to push up this evening.

**Comment by mpash** [comment]
I'm also noticing a large amount of linter errors as well, which kind of takes away from the actual lints I need to fix. Some clean up would be much appreciated! 💃

**Comment by crodriguez1a** [comment]
@mpash Clean up is certainly long over due. I'll begin work on that this week and deploy an update by the weekend.

**Comment by mpash** [comment]
Whooo! Thanks :D

**Comment by jbailey4** [comment]
@crodriguez1a I've started this effort as well. Lets sync up offline and divide and conquer.

**Comment by crodriguez1a** [comment]
@jbailey4 Okay, cool. I'll push up what I have. I replaced jshint with eslint, updated ember suave, and fixed a lot of the minor ones.

**Comment by jbailey4** [comment]
@crodriguez1a Sounds good. Once https://github.com/open-tux/ember-bulma/pull/48 is merged these linting errors should go away.

**Comment by rtablada** [comment]
@crodriguez1a Now that the linting errors have been fixed, can we have a new version cut? 1.5 was published right before the linting fix and still is causing errors :(

**Comment by alexdiliberto** [comment]
@rtablada 0.1.5 should have all the fixes https://github.com/open-tux/ember-bulma/compare/0.1.5...master
